catalog.tests.bom

```diff
@@ -0,0 +1,150 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  items:
+  - id: mysql-node-tests
+    version: 0.10.0-SNAPSHOT
+    itemType: template
+    iconUrl: http://www-jp.mysql.com/common/logos/mysql-logo.svg
+    name: MySql Node tests
+    description: Test that MySql Node is running correctly
+    license_code: Apache-2.0
+    license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+    item:
+      services:
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: Config Keys Tests
+        targetId: mysqlnode
+        brooklyn.children:
+        - type: brooklyn-mysql-node
+          id: mysqlnode
+          brooklyn.config:
+            datastore.creation.script.contents: |
+              CREATE DATABASE temporary;
+              USE temporary;
+              CREATE TABLE users (name VARCHAR(20), owner VARCHAR(20), creation_date DATE, updated_date DATE);
+              INSERT INTO users VALUES ('mysql', 'mysql', CURDATE(), CURDATE())
+            mysql.port: 6789
+
+        # Sensors tests
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: Check server is up
+          target: $brooklyn:component("mysqlnode")
```

Comment:
`component` is deprecated since `0.7.0`. We should really start logging some warnings.
Use `entity` instead.